lib/assets/javascripts/attachinary.js.coffee

```diff
@@ -61,6 +66,9 @@
         headers: {"X-Requested-With": "XMLHttpRequest"}
         dropZone: @config.dropZone || @$input
         sequentialUploads: true
+        disableImageResize: false
```

Comment:
Can we add an parameter to turn image compression on and off? Would be good if we could specify the quality and if it's not there it just doesn't compress.